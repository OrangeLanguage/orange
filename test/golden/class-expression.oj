def point class Point(x, y)
