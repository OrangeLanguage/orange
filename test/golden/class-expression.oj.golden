function _Point(x, y) {
  this.x = x;
  this.y = y;
};
function Point(_handle, x, y) { return new _Point(x(), y()); };
const point = Point;
point